docs/conf/portland/2024/lightning-talks.rst

```diff
@@ -4,11 +4,106 @@
 Lightning Talks
 ===============
 
-{% include "conf/events/lightning-talks.rst" %}
+A lightning talk is a short talk, up to 5 minutes, where you share an idea, concept, or a bit of information you find interesting.
```

Comment:
Why are we moving this back into the conf dir?

Reply:
Because trying to manage edits with a bunch of includes is really difficult.

Reply:
Agreed, but the includes are there for a reason. Especially if we're doing both ways, and across three conferences. I'd rather not revert here, how about just documenting where things are inline? But if we are are we doing it for all 2024 confs? And then how are we updating next year? 

CC @mxsasha for comment here as well.

Reply:
@ericholscher happy to decouple this from merging the PR if you want to merge

Reply:
Yea, I think we can look at backporting changes once we've done our edits, but it's a bit harder because we're mixing "evergreen" and conference-specific content in ordering, because it reads nicer :/